internal/tools/website-scaffold/main_test.go

```diff
@@ -86,17 +87,19 @@ A 'block3' block supports the following:
 				Required: true,
 			},
 			"foo_id": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice([]string{"foo"}, false),
 			},
 			"block2": {
 				Type:     schema.TypeList,
 				Required: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"nest_attr2": {
-							Type:     schema.TypeString,
-							Optional: true,
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringInSlice([]string{"foo", "bar"}, false),
```

Comment:
since we're supporting it above, can we add a test above for the `"Foo", "Bar" and "Bar"` case?

Reply:
i have updated this PR to add a new test-case, but i wonder if my understad is correct... so could you please have a look at it again 💭